tendrl/gluster_integration/objects/node_context/__init__.py

```diff
@@ -37,7 +37,7 @@ def _create_node_id(self, node_id=None):
         with open(local_node_context, 'wb+') as f:
```

Comment:
If there is no way to load `tendrl_context` from local, there is no way we get the details after re-start of `tendrl-gluster-integration` and as mentioned above all the cluster sync details would be wrong.